menderqa_pipeline_helpers.go

```diff
@@ -9,8 +9,21 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+func getLatestIntegrationRelease(number int, conf *config) ([]string, error) {
+	cmd := fmt.Sprintf("git for-each-ref --sort=-creatordate --format='%%(refname:short)' 'refs/tags' "+
+		"| sed -E '/(^[0-9]+\\.[0-9]+)\\.[0-9]+$/!d;s//\\1.x/' | uniq | head -n %d | sort -n -r", number)
+	c := exec.Command("sh", "-c", cmd)
+	c.Dir = conf.integrationDirectory + "/extra/"
+	version, err := c.Output()
+	if err != nil {
+		err = fmt.Errorf("getLatestIntegrationRelease: Error: %v (%s)", err, version)
+	}
+	versionStr := strings.TrimSpace(string(version))
+	return strings.SplitN(versionStr, "\n", -1), err
+}
+
 func getServiceRevisionFromIntegration(repo, baseBranch string, conf *config) (string, error) {
-	c := exec.Command("release_tool.py", "--version-of", repo, "--in-integration-version", baseBranch)
+	c := exec.Command("python3", "release_tool.py", "--version-of", repo, "--in-integration-version", baseBranch)
```

Comment:
Question:
Why was this needed?

Reply:
Because it doesn't work if you don't have "." in your PATH (like on my Mac OS).
Unit tests will fail because release_tool.py is not executable directly. It's either "./release_tool.py" or "python3 release_tool.py".